bpf/Makefile

```diff
@@ -91,10 +91,13 @@ MAX_BASE_OPTIONS += -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 \
 	-DENABLE_EXTERNAL_IP=1 -DENABLE_NODEPORT_ACCELERATION=1 \
 	-DENABLE_SESSION_AFFINITY=1 -DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_DSR=1 \
 	-DENABLE_HOSTPORT=1 -DENABLE_LOADBALANCER -DENABLE_DSR_HYBRID=1 \
-	-DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_IPV4_FRAGMENTS=1 -DENABLE_CUSTOM_CALLS=1
+	-DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_IPV4_FRAGMENTS=1
 ifeq ("$(KERNEL)","netnext")
+# We need to define ENABLE_HOST_SERVICES_FULL to work around complexity issue #14234.
+# We define ENABLE_CUSTOM_CALLS only for net-next (vs. net-next and 4.19) to work
+# around program size issue #15539.
 MAX_BASE_OPTIONS += -DENABLE_TPROXY=1 -DENABLE_REDIRECT_FAST=1 -DENABLE_BANDWIDTH_MANAGER=1 \
-	-DENABLE_HOST_SERVICES_FULL=1 # Required to work around complexity issue #14234.
+	-DENABLE_HOST_SERVICES_FULL=1 -DENABLE_CUSTOM_CALLS=1
```

Comment:
`ENABLE_CUSTOM_CALLS` doesn't exist on 1.9, it's a 1.10 feature. We can drop commit `bpf/Makefile: Disable custom calls on 4.19`.